auslib/test/blobs/test_apprelease.py

```diff
@@ -2883,6 +2883,275 @@ def testSchema7OptionalAttributes(self):
+                "l": {}
+            }
+        }
+    }
+}
+"""))
+        self.blobH2 = ReleaseBlobV8()
+        self.blobH2.loadJSON("""
+{
+    "name": "h2",
+    "schema_version": 8,
+    "hashFunction": "sha512",
+    "appVersion": "31.0",
+    "displayVersion": "31.0",
+    "detailsUrl": "http://example.org/details/%LOCALE%",
+    "actions": "silent",
+    "openURL": "http://example.org/url/%LOCALE%",
+    "notificationURL": "http://example.org/notification/%LOCALE%",
+    "alertURL": "http://example.org/alert/%LOCALE%",
+    "showPrompt": false,
+    "showNeverForVersion": true,
+    "promptWaitTime": 12345,
+    "backgroundInterval": 123,
+    "binTransMerkleRoot": "merkle_root",
+    "binTransCertificate": "cert",
+    "binTransSCTList": "sct_list",
+    "binTransIssuerDigestSPKI": "spki",
+    "fileUrls": {
+        "c1": {
+            "partials": {
+                "h1": "http://a.com/h1-partial.mar"
+            },
+            "completes": {
+                "*": "http://a.com/complete.mar"
+            }
+        },
+        "*": {
+            "partials": {
+                "h1": "http://a.com/h1-partial-catchall"
+            },
+            "completes": {
+                "*": "http://a.com/complete-catchall"
+            }
+        }
+    },
+    "platforms": {
+        "p": {
+            "buildID": 50,
+            "OS_FTP": "p",
+            "OS_BOUNCER": "p",
+            "locales": {
+                "l": {
+                    "partials": [
+                        {
+                            "filesize": 8,
+                            "from": "h1",
+                            "hashValue": "9",
+                            "binTransInclusionProof": """ +
+                             ('''"''' +
+                              "834charpartialsproof834charpartialsproof834charpartialsproof834charpartialsproo"
+                              "f834charpartialsproof834charpartialsproof834charpartialsproof834charpartialspro"
+                              "of834charpartialsproof834charpartialsproof834charpartialsproof834charpartialspr"
+                              "oof834charpartialsproof834charpartialsproof834charpartialsproof834charpartialsp"
+                              "roof834charpartialsproof834charpartialsproof834charpartialsproof834charpartials"
+                              "proof834charpartialsproof834charpartialsproof834charpartialsproof834charpartial"
+                              "sproof834charpartialsproof834charpartialsproof834charpartialsproof834charpartia"
+                              "lsproof834charpartialsproof834charpartialsproof834charpartialsproof834charparti"
+                              "alsproof834charpartialsproof834charpartialsproof834charpartialsproof834charpart"
+                              "ialsproof834charpartialsproof834charpartialsproof834charpartialsproof834charpar"
+                              "tialsproof834charpartialsproof834charpartial" + '''"''') + """
```

Comment:
I think there's some unnecessary punctuation and operators that make this a bit tough to read. How about writing it like this instead?:
```
                            "binTransInclusionProof":
""" +  '"834charpartialsproof834charpartialsproof834charpartialsproof834charpartialsproo'
       'f834charpartialsproof834charpartialsproof834charpartialsproof834charpartialspro'
       'of834charpartialsproof834charpartialsproof834charpartialsproof834charpartialspr'
       'oof834charpartialsproof834charpartialsproof834charpartialsproof834charpartialsp'
       'roof834charpartialsproof834charpartialsproof834charpartialsproof834charpartials'
       'proof834charpartialsproof834charpartialsproof834charpartialsproof834charpartial'
       'sproof834charpartialsproof834charpartialsproof834charpartialsproof834charpartia'
       'lsproof834charpartialsproof834charpartialsproof834charpartialsproof834charparti'
       'alsproof834charpartialsproof834charpartialsproof834charpartialsproof834charpart'
       'ialsproof834charpartialsproof834charpartialsproof834charpartialsproof834charpar'
       'tialsproof834charpartialsproof834charpartial"' + """
                        }

```

Note that the parenthesis are unnecessary, and you can include the double quotes with the value by using single quotes around them instead of double.